Sources/FixedSizeStackable.swift

```diff
@@ -0,0 +1,48 @@
+//  Copyright © 2016 SEEK. All rights reserved.
+//
+
+import UIKit
+
+open class FixedSizeStackable: StackableItem {
+    var view: UIView
+    var width: CGFloat?
+    var height: CGFloat?
+    
+    var size: CGSize {
+        let width = self.width ?? view.intrinsicContentSize.width
+        let height = self.height ?? view.intrinsicContentSize.height
+        return CGSize(width: width, height: height)
+    }
+    
+    public init(view: UIView, width: CGFloat, height: CGFloat) {
+        self.view = view
+        self.width = width
+        self.height = height
+    }
+    
+    public init(view: UIView, width: CGFloat) {
+        self.view = view
+        self.width = width
+    }
+    
+    public init(view: UIView, height: CGFloat) {
+        self.view = view
+        self.height = height
+    }
+    
+    public convenience init(view: UIView, size: CGSize) {
```

Comment:
Do you think it might make a bit more sense to a user if you only had say
```
public convenience init(view: UIView, size: CGSize)
```
and
```
public convenience init(view: UIView)
```

So that you dont have to read code to understand where the size comes from?  Are their situations where you would get the height from fixed and then width from intrinsic?

Reply:
I wanted to offer a user the flexibility with all the options. I'm not sure of a use case off the top of my head. Maybe this class doesn't need to use a view's intrinsicContentSize at all, just always used a fixed size. I'm not fussed with that option.

Reply:
I'm removing intrinsicSize from this class altogether just to make it simpler like you've suggested